Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -431,3 +445,18 @@ import BraintreeDataCollector
         completion(nil, BTPayPalError.canceled)
     }
 }
+
+extension BTPayPalClient: BTAppContextSwitchClient {
+    /// :nodoc:
+    @_documentation(visibility: private)
+    @objc public static func handleReturnURL(_ url: URL) {
+        payPalClient?.handleOpen(url)
+        BTPayPalClient.payPalClient = nil
+    }
+
+    /// :nodoc:
+    @_documentation(visibility: private)
+    @objc public static func canHandleReturnURL(_ url: URL) -> Bool {
+        url.scheme == "https" && (url.path.contains("cancel") || url.path.contains("success"))
```

Comment:
Another thought - might be more readable to keep this similar to Venmo which has a separate class (`BTVenmoAppSwitchReturnURL`) for the `isValid()` logic & eventual param parsing? So adding something like a `BTVeniceAppSwitchReturnURL`

Reply:
I'm down, updated here (I opted to call it `BTPayPalAppSwitchReturnURL` since I assume `Venice` is somewhat internal): https://github.com/braintree/braintree_ios/pull/1218/commits/4b3954130e9b9d91a2718efd1cd82ec6a9548aa5